cluster/gce/trusty/helper.sh

```diff
@@ -18,17 +18,45 @@
 
 # The configuration is based on upstart, which is in Ubuntu up to 14.04 LTS (Trusty).
 # Ubuntu 15.04 and above replaced upstart with systemd as the init system.
-# Consequently, the configuration cannot work on these images.
+# Consequently, the configuration cannot work on these images. In release-1.2 branch,
+# GCI and Trusty share the configuration code. We have to keep the GCI specific code
```

Comment:
Why do we have to worry about keeping this code separate on the master branch? Is it just to make cherry picks easier?